crates/configure-holochain/src/lib.rs

```diff
@@ -124,15 +125,24 @@ async fn install_ui(happ: &Happ, config: &Config) -> Result<()> {
 }
 
 pub async fn update_host_jurisdiction_if_changed(config: &Config) -> Result<()> {
-    if std::env::var("IS_INTEGRATION_TEST")? == "TRUE" {
-        // set in ../tests/integration.rs and ../../holo_happ_manager/tests/integration.ts
-        return Ok(());
+    match std::env::var("IS_INTEGRATION_TEST") {
+        Ok(is_integration_test) => {
+            if is_integration_test == "TRUE" {
+                // set in ../tests/integration.rs and ../../holo_happ_manager/tests/integration.ts
+                return Ok(());
+            }
+        }
+        Err(_) => {} // if we failed to find the env var, we can just carry on
     }
 
     // get current jurisdiction in hbs
-    let hbs_jurisdiction = hpos_holochain_api::get_jurisdiction()
-        .await
-        .context("failed to get jurisdiction from hbs")?;
+    let hbs_jurisdiction = match hpos_holochain_api::get_jurisdiction().await {
+        Ok(hbs_jurisdiction) => hbs_jurisdiction,
+        Err(e) => {
+            debug!("Failed to get jurisdiction from hbs {}", e);
+            return Ok(());
```

Comment:
This will still not let you update jurisdiction, it will just end the update, right?

Reply:
Right. If we can't get the jurisdiction from hbs then we skip the update, because we won't know what to update *to*

Reply:
Ahh, I miss-read it